### DISCUSSION TITLE
What is score in the search results?

### DISCUSSION THREAD

**Discussion by rzw** [zim-desktop-wiki/zim-desktop-wiki#2695] [Q&A]
I couldn't find any documentation about the score column. After a search I had score equal to one everywhere despite the fact that the number of occurences was very different from page to page. Could somebody please explain? Thanks!

**Comment by jaap-karssenberg** [comment]
It is a count except that an occurrence in e.g. a title counts as more than
1 points assuming it is more important than an occurrence somewhere in a
paragraph. Need to check the code for the exact rules.

Op wo 6 nov. 2024 20:30 schreef rzw ***@***.***>:

**Reply by rzw** [reply]
Thanks for your quick answer. Yes, a count is what I had expected. But as described, it wasn't what I saw.

The reason is now clear: When searching for a tag the score is always one. I guess the underlying assumption is that only pages are tagged. But activities/tasks can of course be tagged as well. If there are several tasks with the same tag on a page, this page nonetheless has score one. This is no big deal but not intuitive. If the current behaviour is intentional, fine. If not, I guess this is a bug report.

**Comment by jaap-karssenberg** [comment]
It is intentional, but I'm open to improvements

Op do 7 nov. 2024 20:27 schreef rzw ***@***.***>:

**Reply by rzw** [reply]
It's fine. Just good to know. One can always search without "@". And apart from that I confess that I frequently use dnGrep for search. In Zim it's in my custom tools.

Thanks for the program and your efforts!
